docs/resources/oceancd_verification_template.md

```diff
@@ -0,0 +1,246 @@
+        * `new_relic`    - (Optional) The New Relic provider.
+            * `profile` - (Optional) The name of the secret holding NR account configuration.
+            * `new_relic_query`   - (Required) A raw newrelic NRQL query to perform.
+        * `prometheus`    - (Optional) The Prometheus provider.
+            * `new_relic_query`   - (Required) A request for information retrieved from Prometheus.
+        * `cloud_watch`    - (Optional) The CloudWatch provider.
+            * `duration`   - (Optional) The window of time we are looking at in CloudWatch.
+            * `metric_data_queries` - (Required) The metric queries to be returned. A single MetricData call can include as many as 500 MetricDataQuery structures. Each of these structures can specify either a metric to retrieve, a Metrics Insights query, or a math expression to perform on retrieved data.
+                * `expression` - (Optional) This field can contain either a Metrics Insights query, or a metric math expression to be performed on the returned data. Within one metricdataquery object, you must specify either expression or metric stat but not both.
+                * `id` - (Required) The response ID.
+                * `label` - (Optional) A human-readable label for this metric or expression. If the metric or expression is shown in a CloudWatch dashboard widget, the label is shown
+                * `period` - (Optional) The granularity, in seconds, of the returned data points.
+                * `return_data` - (Optional) This option indicates whether to return the timestamps and raw data values of this metric.If you are performing this call just to do math expressions and do not also need the raw data returned, you can specify False .
+                * `metric_stat` - (Optional) The metric to be returned, along with statistics, period, and units. Use this parameter only if this object is retrieving a metric and not performing a math expression on returned data.
+                    * `metric_period` - (Optional) The granularity, in seconds, of the returned data points.
+                    * `stat` - (Optional) The statistic to return. It can include any CloudWatch statistic or extended statistic.
+                    * `unit` - (Optional) This defines what unit you want to use when storing the metric.
+                    * `metric` - (Optional) The metric to be returned, along with statistics, period, and units. Use this parameter only if this object is retrieving a metric and not performing a math expression on returned data.Within one metricdataquery object, you must specify either expression or metric stat but not both.
+                        * `metric_name` - (Required) The namespace of the metric.
+                        * `namespace` - (Optional) The name of the metric.
+                        * `dimensions` - (Optional) A dimension is a name/value pair that is part of the identity of a metric.
```

Comment:
add below sentence to the description
You can assign up to 30 dimensions to a metric.

Reply:
changed